tests/e2e/utils/response_evaluation.py

```diff
@@ -29,6 +30,23 @@ def __init__(self, eval_args, api_client):
         with open("tests/test_data/question_answer_pair.json") as qna_f:
             self._qa_pairs = json.load(qna_f)["evaluation"]
 
+    def _get_api_response(self, question, provider, model):
+        """Get api response for a question/query."""
+        response = self._api_client.post(
+            "/v1/query",
+            json={
+                "query": question,
+                "provider": provider,
+                "model": model,
+            },
+            timeout=LLM_REST_API_TIMEOUT,
+        )
+        if response.status_code != requests.codes.ok:
```

Comment:
`assert response.status_code == requests.codes.ok`

Reply:
you mean that something from `tests/e2e` will be used from other code?

Reply:
you are right, because it is possible to turn off assertions, it is not a good idea to have them in real code (just in tests). If that's ^ your reason I'm ok with the code ;)